app/build.gradle

```diff
@@ -405,7 +405,10 @@ dependencies {
     androidTestImplementation Deps.mockwebserver
     testImplementation Deps.mozilla_support_test
     testImplementation Deps.androidx_junit
-    testImplementation Deps.robolectric
+    testImplementation (Deps.robolectric) {
+        exclude group: 'org.apache.maven'
+    }
+    testImplementation 'org.apache.maven:maven-ant-tasks:2.1.3'
```

Comment:
Do you still need `maven-ant-tasks`? Is it for `flank`?

Reply:
> Do you still need `maven-ant-tasks`? Is it for `flank`?

This was a change @jonalmeida had us add to fix these dependency issues when we implemented the same for a-c and r-b.  No sure if it's specifically needed for flank or for espresso, but I tried removing it for fenix and get the same errors we saw before (see below) so we'll leave it in place

```> Task :architecture:bundleLibResDebug
> Task :architecture:bundleLibRuntimeDebug

> Task :app:testX86DebugUnitTest
Calling mockable JAR artifact transform to create file: /root/.gradle/caches/transforms-2/files-2.1/2f1d6a0768c5bd92799d46dc19317d53/android.jar with input /opt/android-sdk/platforms/android-28/android.jar

org.mozilla.fenix.components.StoreProviderTest > factory returns store provider FAILED
    java.lang.NoClassDefFoundError
        Caused by: java.lang.ClassNotFoundException

org.mozilla.fenix.components.StoreProviderTest > get only calls createStore if needed FAILED
    java.lang.NoClassDefFoundError

org.mozilla.fenix.components.StoreProviderTest > get returns store FAILED
    java.lang.NoClassDefFoundError

org.mozilla.fenix.components.metrics.PerformedS
```